packages/build-utils/src/lambda.ts

```diff
@@ -47,6 +48,7 @@ interface GetLambdaOptionsFromFunctionOptions {
 
 export class Lambda {
   type: 'Lambda';
+  operationType?: string; // defaults to "API" (or "ISR" for prerenders)
```

Comment:
> (or "ISR" for prerenders)

Should we set this in the `Prerender` constructor?

Reply:
Should the type be more strict to avoid typos?

```ts
operationType?: 'API' | 'ISR'
```